configs:
  mode:
    # Dual-Vision Modes ("dv_usb", "dv_ids")
    # Single-Vision Modes ("sv_offImg", "sv_offImgUV", "sv_offVid", "sv_usbUv", "sv_usbIr", "sv_rs")
    runner: "ids"
  sensor:
    general:
      fpsBoost: true
      brightness:
        alpha: 1.0
        beta: 0.0
    ids:
      exposureTime: 20000
      roi:
        cap1:
          x: 520
          y: 300
          width: 976
          height: 900
        cap2:
          x: 480
          y: 284
          width: 976
          height: 900
  algorithm:
    process:
      alignment:
        matchRate: 0.4
        usePreset: true
        maxFeatures: 500
      channel: "g" # ["b", "g", "r", "all"]
      subtractRL: true
      colorRange:
        hsv_green:
          lower: [35, 120, 50] # [35, 120, 50]
          upper: [85, 255, 255] # [85, 255, 255]
    postprocess:
      threshold:
        size: 50
        method: "binary" # ["binary", "otsu", "both"]
      invertBinary: true
      erosionKernel: 1
      gaussianKernel: 1
  marker:
    structure:
      size: 0.1
    detection:
      dictionary: "DICT_ARUCO_ORIGINAL"
